symforce/codegen/backends/rust/templates/function/FUNCTION.rs.jinja

```diff
@@ -0,0 +1,20 @@
+{# ----------------------------------------------------------------------------
+ # SymForce - Copyright 2022, Skydio, Inc.
+ # This source code is under the Apache 2.0 license found in the LICENSE file.
+ # ---------------------------------------------------------------------------- #}
+
+{%- import "../util/util.jinja" as util with context -%}
+
+
+mod {{ spec.namespace }} {
```

Comment:
The module needs to be made public otherwise the generated function is not accessible.
```suggestion
pub mod {{ spec.namespace }} {
```
I wonder though, is it needed to add a nested module at all? The file itself already creates a module so this defines the function at `outer_module::function_name::sym::function_name`.

 I'm not that familiar with symforce code gen but could a namespace span multiple files that together form a module?

Reply:
hmm I'm not sure. I was really trying to just match the cuda / c++ version here :shrug:  Personally I don't think the mod / namespace was needed but I'm not aware of all the constraints for symforce.